cards.json

```diff
@@ -90,6 +90,11 @@
         "image": "assets/Images/calculator.jpg",
         "link": "Public/calculator.html"
     },
+    {
```

Comment:
Add in alphabetical order & also add your project in README. ANswer me, If you can create a README about this project.

Reply:
Yeah I will add in alphabetic order and about readme I will try to add and tell u if I cannot add.

Reply:
okh I will, but give it a try once

Reply:
Sure do give me 10min I will make a readme having dinner now.

Reply:
No hurry, do it anytime or day. SSOC is 3 month long. :)

Reply:
But I have a doubt like how can add ReadMe for this now?